### DISCUSSION TITLE
Location Notes Don't Display

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#814] [Q&A] [answered]
Hello - Seems like location notes (eg behind bleachers) used to show. Now they don't? Or is there a setting I need to change?

thanks

Roger P

**Comment by joshreisner** [comment, marked as answer]
i just entered location notes on our demo site and i see it showing up: https://code4recovery.org/meetings/carcassonne-monday-evening-meeting

could you share a link to the page on your site? are the changes published?

**Reply by DCO84770** [reply]
This meeting has location notes in the CSV uploaded, but the notes do not show in the meetings admin:
https://dixieaaoffice.com/meetings/?weekday=4&meeting=hurricane-firehouse-group
Seems like the location notes are being stripped off after upload.
Attached is our csv.
[2022-05-20-DCO-meeting-list-01.csv](https://github.com/code4recovery/12-step-meeting-list/files/8783468/2022-05-20-DCO-meeting-list-01.csv)

**Reply by joshreisner** [reply]
i think the issue is that the plugin creates a single location per address. so some of the records in your CSV for "35 State St., Hurricane, UT 84737, USA" have location notes and some do not. this is also true for things like location name.

sometimes it's easier to edit the meetings directly in the plugin -- edits to location notes will apply to all meetings at that location.

if the notes need to vary per meeting, like meeting in different rooms, then those belong in the (meeting) notes field

**Reply by DCO84770** [reply]
Thank You. I moved the items in question to the notes field from the location notes field. Now they show up fine.
Roger
